app/models/sail/setting.rb

```diff
@@ -3,6 +3,7 @@
 require 'fugit'
 
 module Sail
+  # Setting
```

Comment:
Any reason for this comment?

Reply:
@vinistock 
I have enabled `rubocop` in my Sublime editor to ensure I follow Ruby Style Guides all the time. This was a warning from Rubocop so just added a dummy comment for class.